h/api/models/__init__.py

```diff
@@ -10,3 +10,7 @@
 direct to the submodules of this package, but rather through the helper
 functions in `h.api.storage`.
 """
+from h.api.models.elastic import Annotation, Document
```

Comment:
This looks like a rebase error. These are deliberately gone on master so that nobody tries to use them while we migrate to Postgres.